src/pymediainfo/__init__.py

```diff
@@ -173,59 +170,64 @@ def __init__(self, xml: str, encoding_errors: str = "strict"):
         for xml_track in xml_dom.iterfind(xpath):
             self.tracks.append(Track(xml_track))
 
-    def _tracks(self, track_type: str) -> List[Track]:
+    def __eq__(self, other: object) -> bool:
```

Comment:
Why did you move this function after __init__? Is __init__ always supposed to be first?

Reply:
No, I think I moved it at some point because it was easier for.me

Reply:
If we can reduce the diff by not moving it, it'd be nice. I'll do it when I start editing your commits if you agree.

Reply:
Perfect